algorithms/scaling/scaling_options.py

```diff
@@ -194,6 +194,10 @@
       .help = "Cutoff z-score value for identifying outliers based on their
                normalised deviation within the group of equivalent reflections"
       .expert_level = 1
+    emax = 4
```

Comment:
Is this default appropriate for scaling? I note that AIMLESS has a default cutoff of EMAX=10 for acentric reflections:
http://legacy.ccp4.ac.uk/html/aimless.html

Reply:
Updated now to 10 based on precedent set by AIMLESS 👍

Reply:
I did use a default of `emax=4` (based on trial and error) in the Wilson outlier rejection I added to resolution analysis (https://github.com/dials/dials/pull/1580), but I think it is probably approriate for scaling to have a more conservative cutoff for outlier rejection than resolution analysis. There are more details on the choice in AIMLESS in section 5.3 of [Evans, P. (2006). Acta Cryst. D62, 72-82](https://journals.iucr.org/d/issues/2006/01/00/ba5084/):

> Spurious observations arising from ice or salt spots are often very large and may be rejected if they have an intensity much larger than would be expected (Read, 1999). This test is performed on the normalized amplitudes E, normalized as a function of resolution such that 〈E2〉 = 1. An E of > 4 is very unlikely, but because of the errors in normalization, particularly at low resolution where the mean intensity is changing rapidly with resolution and there are relatively few reflections, or with anisotropic data, it is better to reject only observations with E > 8–10.